tests/python/pants_test/source/test_filespec.py

```diff
@@ -47,27 +62,27 @@ def test_matches_single_star_4_neg(self):
     self.assert_rule_match('foo*/bar', ('foofighters/baz/bar',), negate=True)
 
   def test_matches_double_star_0(self):
-    self.assert_rule_match('**', ('a/b/c', 'a'))
+    self.assert_rule_match('**', ('a/b/c', 'b'))
```

Comment:
Why are these results different?

Reply:
This changed because when it comes to real files on disk, both `a` and `a/b/c` cannot be files... `a` would need to be a directory. There is at least one other change for this reason in here.